Lib/test/test_pprint.py

```diff
@@ -1123,6 +1124,217 @@ def test_user_string(self):
+        baz = types.SimpleNamespace(x=321, y="string",
+                                    d={"foo": True, "bar": "baz"})
+        dummy_namespace = types.SimpleNamespace(foo="bar", bar=42, baz=baz)
+        dummy_defaultdict = collections.defaultdict(list)
+        dummy_defaultdict["foo"].append("bar")
+        dummy_defaultdict["foo"].append("baz")
+        dummy_defaultdict["foo"].append("qux")
+        dummy_defaultdict["bar"] = {"foo": "bar", "baz": None}
+        dummy_counter = collections.Counter()
+        dummy_counter.update("foo")
+        dummy_counter.update({ "bar": 5, "baz": 3, "qux": 10 })
+        dummy_chainmap = collections.ChainMap({"foo": "bar"}, {"baz": "qux"},
+                                              {"corge": dummy_dict},)
+        dummy_chainmap.maps.append({"garply": "waldo"})
+        dummy_deque = collections.deque(maxlen=10)
+        dummy_deque.append("foo")
+        dummy_deque.append(123)
+        dummy_deque.append(dummy_dict)
+        dummy_deque.extend(dummy_list)
+        dummy_deque.appendleft(dummy_set)
+
+        class DummyUserDict(collections.UserDict):
+            """A custom UserDict with some extra attributes"""
+
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                self.access_count = 0
+        dummy_userdict = DummyUserDict({ "foo": "bar", "baz": 123,
+                                        "qux": {"foo": "bar", "baz": 123},
+                                        "quux": ["foo", "bar", "baz"],
+                                        "corge": 7 })
+        dummy_userdict.access_count = 5
+
+        class DummyUserList(collections.UserList):
+            """A custom UserList with some extra attributes"""
+
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                self.description = "foo"
+        dummy_userlist = DummyUserList(["first", 2, {"key": "value"},
+                                       [4, 5, 6]])
+
+        dummy_samples = {
+            "dummy_dataclass": dummy_dataclass,
+            "dummy_dict": dummy_dict,
+            "dummy_ordered_dict": dummy_ordered_dict,
+            "dummy_list": dummy_list,
+            "dummy_tuple": dummy_tuple,
+            "dummy_set": dummy_set,
+            "dummy_frozenset": dummy_frozenset,
+            "dummy_bytes": dummy_bytes,
+            "dummy_byte_array": dummy_byte_array,
+            "dummy_mappingproxy": dummy_mappingproxy,
+            "dummy_namespace": dummy_namespace,
+            "dummy_defaultdict": dummy_defaultdict,
+            "dummy_counter": dummy_counter,
+            "dummy_chainmap": dummy_chainmap,
+            "dummy_deque": dummy_deque,
+            "dummy_userdict": dummy_userdict,
+            "dummy_userlist": dummy_userlist,
+        }
+        self.assertEqual(pprint.pformat(dummy_samples, width=40, indent=4, block_style=True, sort_dicts=False),
```

Comment:
and let's wrap this line also
```suggestion
        self.assertEqual(pprint.pformat(dummy_samples, width=40, indent=4,
                                        block_style=True, sort_dicts=False),
```